src/background.js

```diff
@@ -1,77 +1,120 @@
     },
     function(sysObject) {
-        //  if Queeg is detected, go to manager like a boss!
-        if (sysObject) {
-            newUrl = sysObject.host + sysObject.manager;
-            newUrl += '?a=resource/update&id=' + sysObject.id;
+      let createTab = false;
+      //  if Queeg is detected, go to manager like a boss!
+      if (sysObject) {
+        newUrl = sysObject.host + sysObject.manager;
+        newUrl += '?a=resource/update&id=' + sysObject.id;
+        createTab = true;
+      } else {
+        // w/o Queeg: Just try to leave manager
+        if (uri == 'manager') {
+          newUrl = base;
+          chrome.browserAction.setTitle({
+            title: "Preview MODX website"
+          });
+          createTab = true;
         } else {
-            // w/o Queeg: Just try to leave manager
-            if (uri == 'manager') {
-                newUrl = base;
-                chrome.browserAction.setTitle({title:"Preview MODX website"});
-            } else {
-                // w/o Queeg: Just try to open manager
-                newUrl = base + '/manager';
-            }
+          // w/o Queeg: Just try to open manager
+          const managerPath = findManagerPath(host, base, (newUrl) => {
+            chrome.tabs.create({
+                url: newUrl,
+                index: tabIndex + 1
+            });
+          });
+          // newUrl = base + '/manager';
         }
+      }
 
+      if (createTab) {
         chrome.tabs.create({
             url: newUrl,
             index: tabIndex + 1
         });
+      }
     });
 });
 
-chrome.runtime.onMessage.addListener(
-    function(message, sender, sendResponse) {
-
-        // Process system object
-        if (message.system) {
-            delete message.system;
+function findManagerPath(hostInput, base, callbackSuccess) {
+  let host = hostInput;
 
-            if (message.published === 0) { badge_color = error_color };
-            if (message.published === 1) { badge_color = success_color };
+  let managerPath = 'manager';
+  let newUrl = base + '/' + managerPath;
 
-            animateFlip(sender.tab.id);
+  host.replace('https://www.');
+  host.replace('http://www.');
+  host.replace('https://');
+  host.replace('http://');
 
-            chrome.browserAction.setBadgeText ({
-                text: message.id.toString(),
-                tabId: sender.tab.id
-            });
-
-            chrome.browserAction.setBadgeBackgroundColor({
-                color: badge_color,
-                tabId: sender.tab.id
-            })
-        } else {
-            // Process content object
-            delete message.system;
-
-            var tooltip = [];
-            for(var key in message) {
-                tooltip.push(key + ": " + message[key]);
-            }
-
-            chrome.browserAction.setTitle({
-                title: tooltip.join('\n'),
-                tabId: sender.tab.id
-            });
+  chrome.storage.sync.get('pathes', function(result) {
```

Comment:
Plural is "paths", not "pathes" ;)

Reply:
Oops, I will fix it, thanks.
It is a first iteration (coding night), later I will refactor.